wolfram.el

```diff
@@ -137,6 +138,21 @@
           (insert-image (create-image data nil t)))
       (kill-buffer buffer))))
 
+(defun wolfram--query-callback (_args)
+  "Callback function to run after XML is returned for a query."
+  (let* ((data (buffer-string))
+         (pods (xml-get-children
+                (with-temp-buffer
+                  (erase-buffer)
+                  (insert data)
+                  (car (xml-parse-region (point-min) (point-max))))
+                'pod)))
+    (if pods                         ;If at least 1 result pod was returned
+        (wolfram--append-pods-to-buffer
+         (wolfram--switch-to-wolfram-buffer)
+         pods)
+      (message "Your WolframAlpha query turned up with 0 results."))))
```

Comment:
When zero results are returned, the buffer still reads:

`# "asdfasdf" (searching)`

Make sure the `(searching)` part is removed.